app/views/static_pages/about.html.erb

```diff
@@ -1,5 +1,5 @@
 <% provide(:title, "About") %>
-
+<p></p>
```

Comment:
@shaniajubelag What is the purpose of the blank element here? Please remove
If you want to add space, just use margin utility classes in bootstrap, e.g. `mt-1`, `mt-2`, `mt-3`, `mt-4`, `mt-5`